esx_service/cli/vmdkops_admin.py

```diff
@@ -420,8 +420,8 @@ def commands():
                     'help': 'Volume to set options for, specified as "volume@datastore".',
                     'required': True
                 },
-                '--tenant': {
-                    'help': 'Name of the tenant the volume belongs to.',
+                '--vm-group': {
```

Comment:
As discussed offline, please run all tests ```make test-all``` with your change and provide fix for any tests referring to older name i.e. ```--tenant```